vendor/bat-native-ads/src/bat/ads/internal/account/deposits/deposits_database_table.cc

```diff
@@ -166,8 +167,9 @@ void Deposits::PurgeExpired(ResultCallback callback) {
       std::bind(&OnResultCallback, std::placeholders::_1, callback));
 }
 
-std::string Deposits::GetTableName() const {
-  return kTableName;
+const std::string& Deposits::GetTableName() const {
+  static const base::NoDestructor<std::string> table_name(kTableName);
```

Comment:
I think that it is not necessary to cache `table_name` here, because we don't have to do anything to get it

Reply:
Fixed